src/plugins/intel_cpu/src/transformations/snippets/normalize_convert_pre_lower.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <openvino/pass/graph_rewrite.hpp>
+
+namespace ov {
+namespace intel_cpu {
+namespace pass {
+
+// CPU-side pass: for every ov::snippets::op::Subgraph, replace any remaining
+// ov::op::v0::Convert / opset1::Convert inside subgraph body with
+// ov::snippets::op::ConvertTruncation right before lowering in CPU pipeline.
```

Comment:
Could you please specify more context in the pass description (That we workaround android specific problem, its details etc.)?